.vscode/launch.json

```diff
@@ -15,32 +15,31 @@
       "name": "Run All Tests",
       "type": "node",
       "request": "launch",
-      "protocol": "inspector",
-      "program": "${workspaceFolder}/node_modules/mocha/bin/_mocha",
-      "args": ["--inspect", "--no-timeouts", "--colors", "test/**/*.test.ts"],
```

Comment:
`_mocha` -> old executable, use `mocha` instead: https://github.com/mochajs/mocha/blob/master/bin/_mocha

`--inspect` implies `timeout=0` so you can debug tests without having mocha timeout after continuing the process.